integrated_channels/degreed2/exporters/content_metadata.py

```diff
@@ -76,8 +75,22 @@ def transform_duration(self, content_metadata_item):
                 return 0
         else:
             return 0
+
         start_date = parse_datetime_handle_invalid(start)
         end_date = parse_datetime_handle_invalid(end)
+        if not start_date or not end_date:
+            LOGGER.error(
+                generate_formatted_log(
+                    self.enterprise_configuration.channel_code(),
+                    self.enterprise_configuration.enterprise_customer.uuid,
+                    None,
+                    None,
+                    'Failed to find valid start/end dates, so duration is going to be set to 0. '
+                    f'Course item was {content_metadata_item} '
+                    f'Parsed Start was: {start_date}, End was: {end_date}'
+                )
+            )
+            return 0
```

Comment:
Yeah Alex and I also thought about this a bit. This is a case where we can't determine duration. so we have a choice. We can always report out a 0, which is simpler in the sense of using the payload/api. And I think it's going to be okay from a Degreed customer perspective to see 0 days? If we feel like that is not acceptable, we can change it not include duration at all. All the other cases already return 0 too so I did this to be consistent there.

Reply:
followed up on slack with Matt. We decided that it's fine for this PR to unblock this way but we also want to think about a way to skip sending 'invalid' content metadata items (this may be a case of that), we will discuss on Monday LMS hour